src/main/java/gomedic/logic/parser/CliSyntax.java

```diff
@@ -5,11 +5,16 @@
  */
 public class CliSyntax {
 
-    /* Prefix definitions */
+    /* Prefix definitions for a person */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
```

Comment:
PREFIX_TAG here is also currently t/, I think it is possible to parse this still, but do you think it is better to change as you also use t/ to differentiate the type of objects added (t/activity).